### ISSUE TITLE
命令EXPLORER ASF 报错

### ISSUE THREAD

**Issue by TT-IPIP** [chr233/ASFEnhance#24]
```
ERROR ASF > OnBotCommand() System.NullReferenceException: Object reference not set to an instance of an object.
at ASFEnhance.Explorer.ReflectionHelper.GetPrivateField[T](Object instance, String fieldname)
at ASFEnhance.Explorer.Command.ResponseExploreDiscoveryQueue(Bot bot)
at ASFEnhance.Explorer.Command.<>c.<ResponseExploreDiscoveryQueue>b__1_1(Bot bot)
at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
at System.Threading.Tasks.Task.WhenAll[TResult](IEnumerable`1 )
at ArchiSteamFarm.Core.Utilities.InParallel[T](IEnumerable`1 tasks)
at ASFEnhance.Explorer.Command.ResponseExploreDiscoveryQueue(String botNames)
at ASFEnhance.ASFEnhance.OnBotCommand(Bot bot, EAccess access, String message, String[] args, UInt64 steamID)
at ArchiSteamFarm.Core.Utilities.InParallel[T](IEnumerable`1 tasks)
at ArchiSteamFarm.Plugins.PluginsCore.OnBotCommand(Bot bot, EAccess access, String message, String[] args, UInt64 steamID)
```

**Comment by SENPAY98K** [comment]
EXPLORER somehow its not working,

**Comment by woctezuma** [comment]
Similar to:
- #22

**Comment by chr233** [comment]
What's ASF's version

**Comment by SENPAY98K** [comment]
Latest version `ASF 5.2.6.3 - win-x64 - 5c7d999`

**Comment by wcwj** [comment]
add   AutoSteamSaleEvent": true,     to  bot json file shoud be ok

**Comment by TheRhanderson** [comment]
Also happening here. It seems ASFEnhance has issues to force the function on multiple accounts.
Using for 2, 3, 4 accounts every few minutes, it works just fine.
Using for 10+ or "ex asf, explorer asf", it fails.

Tested on ASF v.5.2.7.6 and v5.2.6.3

**Comment by Botan626** [comment]
`"DevFeature": true` in ASF.json, but `ACCESSTOKEN [Bots]` command returns 'Unknown command!'.

**Comment by Botan626** [comment]
> ASF 5.2.7.6 - generic + ASFEnhance 1.6.6.622 installed `"DevFeature": true` in ASF.json, but `ACCESSTOKEN [Bots]` command returns 'Unknown command!'.

with ASFEnhance 1.6.7.651 `ACCESSTOKEN [Bots]` says:
> ASFEnhance.DevFeature is disabled, this feature is unavilable

DevFeature is disabled even with `"DevFeature": true` in ASF.json?

**Comment by chr233** [comment]
> > ASF 5.2.7.6 - generic + ASFEnhance 1.6.6.622 installed `"DevFeature": true` in ASF.json, but `ACCESSTOKEN [Bots]` command returns 'Unknown command!'.
> 
> with ASFEnhance 1.6.7.651 `ACCESSTOKEN [Bots]` says:
> 
> > ASFEnhance.DevFeature is disabled, this feature is unavilable
> 
> DevFeature is disabled even with `"DevFeature": true` in ASF.json?
> 
> Edit: there is a typo, should be "unavailable".

![image](https://user-images.githubusercontent.com/13798576/176188720-7b1983ba-aea0-4a24-a6d0-24e987727897.png)

**Comment by chr233** [comment]
> * GetPrivateField

我知道是什么问题了，bot没有开启 autosaleevent 的话 steamevent 读出来是null

**Comment by chr233** [comment]
Fixed，更新到 1.6.7.654 应该可以解决。
